openhexa/sdk/workspaces/workspace.py

```diff
@@ -43,6 +43,10 @@ def _token(self) -> str:
         except KeyError:
             raise WorkspaceConfigError("The workspace token is not available in this environment.")
 
+    @property
+    def _connected(self):
```

Comment:
This addition is weird.

Reply:
When we define a pipeline with a DatasetParam that has default value set, the Parameter class try to validate it by calling the DatasetType.validate (as we have done for the connection) and it raises   
``` python
WorkspaceConfigError( "The workspace slug is not available in this environment." )
``` 

So I've added this as a guard to prevent making request to the API if not available. If you have others suggestions  I'm open to it.

Reply:
Nevermind, I figured it. Needed to add the validate_default